Makefile

```diff
@@ -1,23 +1,48 @@
-DIST=dist/
-APPNAME=halsecur
+DIST := dist
+APPNAME := halsecur
 
-GOLANGCILINT_VERSION=v2.7.2
-GOSEC_VERSION=v2.22.11
-VULNCHECK_VERSION=latest
+GOLANGCILINT_VERSION := v2.7.2
+GOSEC_VERSION := v2.22.11
+VULNCHECK_VERSION := latest
+
+LDFLAGS := -X bisecur/version.Version=?version? -X bisecur/version.BuildDate=?date?
+
+ifeq ($(OS),Windows_NT)
+	SHELL := cmd.exe
+	SHELLFLAGS := /C
+
+	EXE := .exe
```

Comment:
My Linux based tests reveled that `EXE` variable needs to be always defined regardless the OS. I built binary for both Windows and Linux. Because `EXE` variable was not defined, Linux binary was overwritten. When I moved line #14 outside of the `ifeq` statement, I got two separated binaries as expected.

After this, we can merge. Thank you for your work :)

<img width="296" height="43" alt="image" src="https://github.com/user-attachments/assets/a76d7e5d-047e-4ce4-acea-695189880307" />